### PR TITLE
3.0 ru skeleton

### PR DESCRIPTION
**Pull request by Leo240**

### CODE REVIEW

File: ru/index.rst [hunk 1]
```diff
@@ -14,15 +14,15 @@ Welcome
...
     .. image:: /_static/img/read-the-book.jpg
 
-    Enjoy the CakePHP cookbook almost anywhere. Available as both a PDF and
-    EPUB, you can now read it on more devices, as well as offline.
+    Наслаждайтесь книгой рецептов CakePHP почти везде. Она дступна как в формате PDF,
+    так и в EPUB, вы можете теперь читать ее на многих устройствах, даже офлайн.
 
-    - `PDF <../_downloads/en/CakePHPCookbook.pdf>`_
-    - `EPUB <../_downloads/en/CakePHPCookbook.epub>`_
+    - `PDF <../_downloads/ru/CakePHPCookbook.pdf>`_
```
Comment (antograssiot) [inline ru/index.rst]: @markstory maybe you will remember this better than I do but does the pdf generation is working with Russian alphabet ?

Comment (Leo240) [inline ru/index.rst]: I don't know exactly. If not - I will not change links like that one.

**Comment by cake17** [comment]
Thanks @Leo240. I'll try to build ru version and merge the 3.0-ru-skeleton into 3.0 branch

**Comment by Leo240** [comment]
@cake17 I will try to finish the basic doc files in few days

**Pull request merged**